generate-elm/Main.hs

```diff
@@ -6,12 +6,13 @@
 
 module Main where
 
-import Elm (ElmType (..), toElmDecoderSource, toElmEncoderSource, toElmTypeSource)
-import Servant.API ((:>))
-import Servant.Elm (Spec (..), generateElmForAPI, specsToDir)
+import           Elm              (ElmType (..), toElmDecoderSource,
+                                   toElmEncoderSource, toElmTypeSource)
+import           Servant.API      ((:>))
+import           Servant.Elm      (Spec (..), generateElmForAPI, specsToDir)
```

Comment:
You used wrong `.stylish-haskell.yaml` configuration. `three-layer` repository has `.stylish-haskell.yaml`. Probably this happened because `stylish-haskell` was executed not from `three-layer` directory or from any its subdirectory. As you can see, formatting in this file was changed a lot (this alignment and others). And usually this should never happen.